src/main/java/uk/gov/digital/ho/hocs/audit/service/SomuExportService.java

```diff
@@ -177,7 +177,7 @@ private String getCaseTypeCode(String caseType) {
 
     private String[] getHeaders(List<SomuTypeField> somuTypeFields) {
         String[] headers = new String[]{
-                "timestamp", "event", "userId", "caseUuid", "somuItemUuid", "somuTypeUuid"
+                "timestamp", "event", "userId", "caseReference", "somuItemUuid", "somuTypeUuid"
```

Comment:
This no longer follows previous functionality - which was the original intention. 

Unsure if this is a real issue...

Reply:
Reverted to original intention